javascript/chronometer.js

```diff
@@ -1,26 +1,65 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
+
   startClick(callback) {
-    // ... your code goes here
+   this.currentTime = 0;
+   let intervalId = setInterval(() =>{
+     this.currentTime += 1;
+   }, 1000);  
+
+   return this.currentTime;
+
   }
+
   getMinutes() {
-    // ... your code goes here
+   let min = this.currentTime / 60;
+   min = Math.floor(min);
+   Number(min.toFixed(0));
```

Comment:
Math.floor already gives you an integer number, so there is no need to use toFixed(0)